server/src/schema.js

```diff
@@ -1,112 +1,138 @@
-const {gql} = require('apollo-server');
+const { gql } = require("apollo-server");
 
 const typeDefs = gql`
-    scalar Date
-    scalar DateTime
+  scalar Date
+  scalar DateTime
 
-    type Query {
-        # Returns certain size of events after the cursor
-        # Reference:
-        #  https://www.apollographql.com/docs/tutorial/resolvers/#paginated-queries 
-        events(pageSize: Int, after: String): EventConnection!
-        # Return specific event whose id is 'id'.
-        # If not exist, return null
-        event(id: ID!): Event
-        # Return the user currently logged in
-        me: User
-        # Return the user whose id is 'id'.
-        # If not exist, return null
-        user(id: ID!): User
-    }
+  type Query {
+    # Returns certain size of events after the cursor
+    # Reference:
+    #  https://www.apollographql.com/docs/tutorial/resolvers/#paginated-queries
+    events(pageSize: Int, after: Int): EventConnection!
+    # Return specific event whose id is 'id'.
+    # If not exist, return null
+    event(id: ID!): Event
```

Comment:
Why are some comments missing #?

Reply:
They are not comment. It's working code.